ui/src/dashboards/constants/index.js

```diff
@@ -68,6 +68,16 @@ export const generateNewDashboardCell = dashboard => {
   }
 }
 
+export const cloneDashboardCell = (dashboard, cloneCell) => {
+  const newCellY = dashboard.cells
+    .map(cell => cell.y + cell.h)
+    .reduce((a, b) => (a > b ? a : b))
```

Comment:
nice! so it finds the bottom of the bottom-most cell and sets that to be `newCellY`?